canopy_dispatch.ml

```diff
@@ -0,0 +1,108 @@
+open Lwt.Infix
+
+type store_ops = {
+  subkeys : string list -> string list list Lwt.t ;
+  value : string list -> string option Lwt.t ;
+  update : unit -> string list Lwt.t
+}
```

Comment:
this is the piece I'm not happy abou... but I failed to properly construct te `module Store` and pass it to `Canopy_dispatch`...